src/plotSearch/components/PlotSearchListPage.js

```diff
@@ -73,9 +73,9 @@ const visualizationTypeOptions = [
   {value: VisualizationTypes.MAP, label: 'Kartta', icon: <MapIcon className='icon-medium' />},
 ];
 
-type OwnProps = {
+type OwnProps = {|
 
-};
+|};
```

Comment:
I needed to google what the pipes are doing here. But yeah, makes sense now when I know :D